app/src/main/java/org/simple/clinic/editpatient/EditPatientConfig.kt

```diff
@@ -0,0 +1,14 @@
+package org.simple.clinic.editpatient
+
+import org.simple.clinic.remoteconfig.ConfigReader
+import javax.inject.Inject
+
+data class EditPatientConfig(
+    val canDeletePatient: Boolean
+) {
+
+  @Inject
+  constructor(configReader: ConfigReader) : this(
+      canDeletePatient = configReader.boolean("can_delete_patient", false)
```

Comment:
Rename this property to `deletePatientFeatureEnabled` to make it clear it is a feature flag?